src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -19,7 +19,7 @@
                  @(_DebugSymbolsIntermediatePath);
                  $(NonExistentFile);
                  @(CustomAdditionalCompileOutputs)"
-          Returns=""
+          Returns="$(CscCommandLineInvocation)"
```

Comment:
Nitpick: I'd create a new target called GetCommandLineInvocation or something. Entirely 100% happy if that just DependsOnTargets="CoreCompile", but that way the contract is a bit more explicit. If somebody came by and quizzed me asking what CoreCompile returned, I don't think this would be my first guess. (First guess would be the path to the output...)